### PR TITLE
Swapsort algorithm and corresponding tests

### PR DESCRIPTION
**Pull request by chnlmshr**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add SwapSort Algorithm
- [x] Add SwapSort Tests

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file and its corresponding test file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Sorts/SwapSort.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * @description Swap Sort is an algorithm to find the number of swaps required to sort an array.
+    Time complexity of Swap Sort Algorithm is O(nlogn).
+    Auxiliary Space required for Swap Sort Algorithm is O(n).
+ * @param {Integer[]} items - Array of integers
+ * @return {Integer} - Number of swaps required to sort the array.
+ * @see [SwapSort](https://www.geeksforgeeks.org/minimum-number-swaps-required-sort-array/)
+ */
+
+export function swapSort (items) {
+  const sortedArray = items.slice()
```
Comment (appgurueu) [inline Sorts/SwapSort.js:12]: Sorting algorithms usually operate in place (don't copy the array).

File: Sorts/test/SwapSort.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { swapSort } from '../SwapSort'
+
+describe('SwapSort', () => {
```
Comment (appgurueu) [inline Sorts/test/SwapSort.test.js:3]: Please test whether the arrays are sorted properly.

File: Sorts/SwapSort.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/**
+ * @function SwapSort
+ * @description Swap Sort is an algorithm to find the number of swaps required to sort an array.
```
Comment (appgurueu) [inline Sorts/SwapSort.js:3]: Then please rename the function to `minSwapsToSort` or similar

Comment (chnlmshr) [reply Sorts/SwapSort.js:3]: updated

**Review by raklaptudirm — APPROVED** [review]

**Review by raklaptudirm — APPROVED** [review]

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**